cmd/recv.go

```diff
@@ -215,7 +215,7 @@ func recvAction(cmd *cobra.Command, args []string) {
 					}
 
 					dir := filepath.Dir(p)
-					err = os.MkdirAll(dir, 0777)
+					err = os.MkdirAll(dir, 0700)
```

Comment:
Gosec suggests `0750` as that means `read/write/execute` for user and `read/execute` for group and no permissions for other. I think that might make more sense.

Reply:
The reason for choosing `0700` was that since this is a program that is a very privacy oriented file transfer program, it would be prudent choose a more "paranoid" default. For instance, if a recipient got very sensitive files and didn't want another user to access it. As a program, we guarantee such a behaviour instead of relying on the operating system.

Reply:
I know but I think `0700` is too strict given that it doesn't include group permissions. Using `0750` will be just as good but without potential permission issues.